backend/test/controllers/api/UsersTest.scala

```diff
@@ -39,7 +39,7 @@ class UsersTest extends AnyFreeSpec with Matchers with Results with ScalaFutures
 
   "UsersController" - {
     "list partial user information to punters" in {
-      TestSetup(users + (paul -> (UserPermissions.bigBoss, List.empty), barry -> (UserPermissions.default, List.empty)), barry) { (controller, _) =>
```

Comment:
`+` with varargs is deprecated so it's easier just to change the test to include the default set of users automatically (see the bottom of the file)